pysolr.py

```diff
@@ -830,11 +843,17 @@ def search(self, q, search_handler=None, **kwargs):
         response = self._select(params, handler=search_handler)
         decoded = self.decoder.decode(response)
 
-        self.log.debug(
-            "Found '%s' search results.",
-            # cover both cases: there is no response key or value is None
-            (decoded.get("response", {}) or {}).get("numFound", 0),
-        )
+        if decoded.get("grouped"):
```

Comment:
I would defer support for grouped responses to a larger task which would provide more help for them - just changing a log message doesn't seem to add much value.